app/models/manifest.rb

```diff
@@ -12,10 +18,6 @@ def content_field(json_xpath)
     end
   end
 
-  def tracking_number
```

Comment:
if we kept this method (and renamed) we could keep the json validation for tracking number and not worry about setting tracking number field in a callback